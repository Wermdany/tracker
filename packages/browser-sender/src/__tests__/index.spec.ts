import * as modules from "..";

describe("test browser-sender output modules", () => {
  it("should browser-sender output modules", () => {
    expect(Object.keys(modules)).toMatchSnapshot();
  });
});
