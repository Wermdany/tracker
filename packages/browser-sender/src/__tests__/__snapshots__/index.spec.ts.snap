// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test browser-sender output modules should browser-sender output modules 1`] = `
[
  "SenderError",
  "XhrSenderPlugin",
  "createXhrSenderFactory",
  "BeaconSenderPlugin",
  "createBeaconSenderFactory",
  "FetchSenderPlugin",
  "createFetchSenderFactory",
  "useMiddleware",
  "ErrorRetryMiddleware",
  "BatchSendMiddleware",
]
`;
